src/app/shared/toast/toast.component.ts

```diff
@@ -79,6 +110,27 @@ export class ToastComponent implements OnInit, OnDestroy {
     }
   }
 
+  pauseTimer() {
+    this._toasts.cancelToastTimer(this.toast.id)
+  }
+
+  resumeTimer(event: any) {
```

Comment:
When visiting the Go to group page link, or another route, `this._dom.byId("addToGroupDropdown").focus()` will result in a console error.

The very short timeout amount, along with checking for activeAlement is somewhat brittle. Let's talk and brainstorm some options for different solutions.